tests/k8s-azure/manifest/linux.json

```diff
@@ -4,11 +4,11 @@
   "properties": {
     "orchestratorProfile": {
       "orchestratorType": "Kubernetes",
-      "orchestratorRelease": "1.11",
+      "orchestratorVersion": "1.12.0-alpha.1",
       "kubernetesConfig": {
           "useCloudControllerManager": true,
-          "customCcmImage": "gcrio.azureedge.net/google_containers/cloud-controller-manager-amd64:v1.12.0-alpha.0",
-          "customHyperkubeImage": "gcrio.azureedge.net/google_containers/hyperkube-amd64:v1.12.0-alpha.0",
+          "customCcmImage": "gcrio.azureedge.net/google_containers/cloud-controller-manager-amd64:v1.13.0-alpha.0",
+          "customHyperkubeImage": "gcrio.azureedge.net/google_containers/hyperkube-amd64:v1.13.0-alpha.0",
```

Comment:
should we keep those version same with orchestratorVersion?

Reply:
This is image version, should be latest for testing.
orchestratorVersion is acs-engine's config.

Reply:
Thanks